tests/test_config.cpp

```diff
@@ -4,17 +4,25 @@
 
 BOOST_AUTO_TEST_SUITE(ConfigTests)
 
+
 BOOST_AUTO_TEST_CASE(test_loading) {
     // Write out a temporary file
     std::ofstream test_config("/tmp/openrw_test.ini");
     test_config << "[game]\n"
-                << "path=/dev/test" << std::endl;
+                << "\tpath=\t/dev/test\n"
+                << " language = american ;Comment\n"
+                << ";lineComment\n"
+                << "nonexistingkey=somevalue"
+                << std::endl;
 
     GameConfig config("openrw_test.ini", "/tmp");
 
     BOOST_CHECK(config.isValid());
 
     BOOST_CHECK_EQUAL(config.getGameDataPath(), "/dev/test");
+    BOOST_CHECK_EQUAL(config.getGameLanguage(), "american");
+    BOOST_CHECK_EQUAL(config.getInputInvertY(), false);
 }
 
+
```

Comment:
Since this PR adds support for writing configuration to a file/a string, could you add a test to make sure that keeps working until it's used in-game?